cmd/drone-docker/main.go

```diff
@@ -208,6 +208,11 @@ func main() {
 			Usage:  "docker email",
 			EnvVar: "PLUGIN_EMAIL,DOCKER_EMAIL",
 		},
+		cli.StringFlag{
+			Name:   "docker.authconfig",
+			Usage:  "docker json authconfig content",
+			EnvVar: "PLUGIN_AUTHCONFIG,DOCKER_AUTHCONFIG",
```

Comment:
this is more of a small nit, but let's name this `PLUGIN_CONFIG` since technically speaking it is a config file.  The config file includes auth, but it could technically contain other non-auth related configuration parameters.

Reply:
Good Idea